cmd/openauthctl/migrations/000077_api_keys.up.sql

```diff
@@ -0,0 +1,26 @@
+create table api_keys (
+    id uuid not null primary key,
+    organization_id uuid not null references organizations (id) on delete cascade,
+    display_name varchar not null,
+    secret_token_sha256 bytea,
+    secret_token_suffix varchar(4),
```

Comment:
just do `varchar`, Postgres follows the SQL spec on `varchar(n)`, and the spec is not sane